furms-web-ui/src/main/java/io/imunity/furms/ui/components/FurmsSelect.java

```diff
@@ -65,4 +74,20 @@ private void addSeparators(Map<ViewMode, List<FurmsViewUserContext>> data) {
 			}
 		}
 	}
+
+	@Override
+	protected void onAttach(AttachEvent attachEvent) {
+		super.onAttach(attachEvent);
+		UI ui = attachEvent.getUI();
+		broadcasterRegistration = vaadinBroadcaster.register(
+			event -> ui.access(this::reloadComponent)
```

Comment:
Should I use switch here to check type of event? I don't have any good idea.

Reply:
Can you elaborate a bit on what problem is envisioned to be solved by this switch?

Reply:
To chose logic for event type.